homeassistant/components/homeassistant/scene.py

```diff
@@ -76,7 +77,11 @@ def _convert_states(states):
 )
 
 CREATE_SCENE_SCHEMA = vol.Schema(
-    {vol.Required(CONF_SCENE_ID): cv.slug, vol.Required(CONF_ENTITIES): STATES_SCHEMA}
+    {
+        vol.Required(CONF_SCENE_ID): cv.slug,
+        vol.Optional(CONF_ENTITIES, default={}): STATES_SCHEMA,
+        vol.Optional(CONF_SNAPSHOT, default=[]): cv.entity_ids,
```

Comment:
Should we require at least one of the keys `CONF_ENTITIES` and `CONF_SNAPSHOT`?

Reply:
At least one of these is required. But how can you tell that voluptuous?

Reply:
Use `cv.has_at_least_one_key`.

Reply:
Thanks :+1: